dotnet/src/Agents/OpenAI/Extensions/KernelExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.SemanticKernel.Agents.OpenAI.Extensions;
+
+internal static class KernelExtensions
+{
+    /// <summary>
+    /// Retrieve a kernel function based on the tool name.
+    /// </summary>
+    public static KernelFunction GetKernelFunction(this Kernel kernel, string toolName, char delimiter)
```

Comment:
Does this need to be public?

Reply:
The class is scoped `internal` (double checked other file also).  

Do you prefer for the methods to be declared `internal` also?  Scanning the repo, I'm seeing somewhat of a mixed convention:

https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatPromptParser.cs